### ISSUE TITLE
Undesirable gaps when codelens is turned on

### ISSUE THREAD

**Issue by henry-js** [leodevbro/vscode-blockman#34]
When using reference codelens, blocks are not rendered on the reference lines, this results in gaps that are quite jarring:
![image](https://user-images.githubusercontent.com/79054685/126880760-66aa79bb-4403-4d14-8e9d-2738aba770c9.png)

**Comment by leodevbro** [comment]
Yeah, this is not exactly a bug of Blockman, but more of a product of the limitations of VSCode extension API.
the blocks are rendered line by line, not as entire divs, this is because VSCode API has some strange limitations, for example, if any div is higher than line height, then if we scroll to make it half hidden, the other half also disapears, thus behaving with uncomfortable manner.
Maybe I will post a feature request in vscode repo about this limitation.
